src/bootstrap/src/core/builder/tests.rs

```diff
@@ -1793,11 +1793,8 @@ mod snapshot {
                 .path("src/tools/compiletest")
                 .stage(1)
                 .render_steps(), @r"
-        [build] llvm <host>
-        [build] rustc 0 <host> -> rustc 1 <host>
-        [build] rustc 1 <host> -> std 1 <host>
-        [build] rustdoc 1 <host>
-        [doc] Compiletest <host>
+        [build] rustdoc 0 <host>
+        [doc] rustc 0 <host> -> Compiletest 1 <host>
```

Comment:
Question: hm, to clarify, the staging in `./x doc $thing --stage=N` corresponds to "what gets built"

Reply:
Well, for `cargo/x doc`, "what gets built" corresponds to "what gets documented", so yeah.